recipes/gino/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "gino" %}
+{% set version = "0.8.1" %}
+{% set sha256 = "3471e2f5b44f787eff3381d47e3e252ba184749ba2747adca06658d56272e912" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
nit: I think the flavour of the week is to use `{{ PYTHON }}`

Reply:
Ditto here.